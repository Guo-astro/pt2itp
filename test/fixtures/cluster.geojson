{
  "type": "FeatureCollection",
  "features": [
    {
      "type": "Feature",
      "properties": {},
      "geometry": {
        "type": "Point",
        "coordinates": [
          9.51413869857788,
          47.132724392963944
        ]
      }
    },
    {
      "type": "Feature",
      "properties": {},
      "geometry": {
        "type": "Point",
        "coordinates": [
          9.516541957855225,
          47.132724392963944
        ]
      }
    }
  ]
}