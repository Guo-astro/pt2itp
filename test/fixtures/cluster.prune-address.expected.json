[
  {
    "id": "1",
    "text": "Main St",
    "text_tokenless": "Main"
  },
  {
    "id": "2",
    "text": "Pollard Rd",
    "text_tokenless": "Pollard"
  },
  {
    "id": "3",
    "text": "First St",
    "text_tokenless": "First"
  }
]